lib/phlex/sgml.rb

```diff
@@ -222,10 +223,10 @@ def flush
 		# 		render @items
 		def render(renderable, &block)
 			case renderable
-			when Phlex::SGML
+			when Phlex::Renderable
```

Comment:
One problem with this is you could potentially render an unsafe CSV into an HTML view.

Reply:
Actually, that would prevent you from rendering SVGs inside HTMLs. Perhaps it could be guarded by `html_safe?`/`svg_safe?` on the renderable. 🤔